cytominer_database/ingest.py

```diff
@@ -99,12 +99,12 @@ def into(input, output, name, identifier, skip_table_prefix=False):
             #     deprecated, use inspect.signature() or inspect.getfullargspec()
             warnings.simplefilter("ignore", category=DeprecationWarning)
 
-            target = "{}::{}".format(output, name)
+            target = output
```

Comment:
is this line required? can we just do `engine = create_engine(output)` in the next line?

Reply:
Sounds good, will do